templates/pages/admin/form/form_editor.html.twig

```diff
@@ -59,6 +59,7 @@
     action="{{ item.getFormURL() }}"
     data-ajax-submit {# Form will be submitted using AJAX #}
     data-ajax-submit-itemtype="Glpi\Form\Form" {# Target itemtype #}
+    data-track-changes="true"
```

Comment:
Not related specifically to this PR, but maybe this form should also have `data-submit-once` to prevent issues when a user double clicks the submit button(s).

Reply:
It would be nice to have but from my tests `data-submit-once` doesn't seems to work correctly with AJAX forms.